charts/trivy-operator-crds/README.md

```diff
@@ -0,0 +1,56 @@
+# trivy-operator-crds
+
+Helm chart for deploying trivy-operator CRDs.
+
+### How to update the Chart
+
+The Chart has the same version as the `trivy-operator`, try to keep them equal.
+
+CRDs are retrieved from [here](https://github.com/aquasecurity/trivy-operator/tree/main/deploy/helm/crds).
+
+
+To download them, you can use the following script. 
+Be aware to run it where you want the files.
+```
```

Comment:
you the same script as used here instead:
- https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/traefik-crds#how-to-update-the-chart
and put it inside the scripts folder since multiple charts use it